### PR TITLE
[jasmine-given] add fail function to done function definition

### PR DESCRIPTION
**Pull request by shairez**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: 

**Not a URL, but an explanation** - the changes were always there, they just weren't reflected in the type definitions

- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/jasmine-given/index.d.ts [hunk 1]
```diff
@@ -3,9 +3,17 @@
...
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare function Given(func: (done?: () => void) => void): void;
-declare function When(func: (done?: () => void) => void): void;
-declare function Then(func: (done?: () => void) => void): void;
-declare function Then(label: string, func: (done?: () => void) => void): void;
-declare function And(func: (done?: () => void) => void): void;
-declare function Invariant(func: (done?: () => void) => void): void;
+/** Action method that should be called when the async work is complete */
+interface DoneFn extends Function {
```
Comment (sheetalkamat) [inline types/jasmine-given/index.d.ts]: You don't need extends Function clause. Given you have call signature should suffice.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

**Comment by shairez** [comment]
@sheetalkamat changed

File: types/jasmine-given/index.d.ts (hunk 1 above)
Comment (shairez) [reply types/jasmine-given/index.d.ts]: changed

**Comment by shairez** [comment]
@sheetalkamat @RyanCavanaugh 
I submitted the changes
is there anything else I need to do? 

Thanks!

**Review by sheetalkamat — APPROVED** [review]

**Pull request merged**